helm/superset/values.yaml

```diff
@@ -104,7 +104,10 @@ extraConfigMountPath: "/app/configs"
 image:
   repository: apache/superset
   tag: latest
```

Comment:
Given that you are setting the chart app version to 1.1.0, this should be set to the tag of 1.1.0
apache/superset:latest is master

See: https://hub.docker.com/r/apache/superset/tags

Reply:
thanks, edited.